diff_drive_controller/include/diff_drive_controller/diff_drive_controller.h

```diff
@@ -126,10 +126,13 @@ namespace diff_drive_controller
     /**
      * \brief Default wheel speed limiter function, can be overridden
      *        in a controller which extends this one.
-     * \param [in, out] left_velocity  Left wheel velocity (not used)
-     * \param [in, out] right_velocity Right wheel velocity (not used)
+     * \param [in, out] left_command  Commanded left wheel velocity.
+     * \param [in, out] right_command Commanded right wheel velocity.
+     * \param [in]      left_desired  Desired left wheel velocity.
+     * \param [in]      right_desired Desired right wheel velocity.
```

Comment:
Can this comment please be beefed up to explain the difference between "commanded" and "desired" (even just paste in what's in the PR body).

Reply:
Done